docs/manual/java/guide/cluster/PersistentEntity.md

```diff
@@ -69,7 +69,10 @@ The command can be validated before persisting state changes. Use `ctx.invalidCo
 
 @[validate-command](code/docs/home/persistence/Post2.java)
 
-A `PersistentEntity` may also process commands that do not change application state, such as query commands. Such command handlers are registered using `setReadOnlyCommandHandler` of the `BehaviorBuilder`. Replies are sent with the `reply` method of the context that is passed to the command handler function.
+A `PersistentEntity` may also process commands that do not change application state, such as query commands  or commands that are not valid in the entity's currernt state (such as a bid placed after the auction closed). Such command handlers are registered using `setReadOnlyCommandHandler` of the `BehaviorBuilder`. Replies are sent with the `reply` method of the context that is passed to the command handler function.
```

Comment:
hi @TimPigden, I was going to merge and noticed a typo for the word `currernt` (in both files).